### PR TITLE
Add bounded service account key kms id to Sts model

### PR DESCRIPTION
**Pull request by gdbranco**
Related issue: https://issues.redhat.com/browse/SDE-1882

### CODE REVIEW

**Comment by gdbranco** [comment]
@tzvatot @zgalor PTAL

File: model/clusters_mgmt/v1/sts_type.model [hunk 1]
```diff
@@ -45,4 +45,7 @@ struct STS {
 
 	// Private key used to sign the service account token and allow cluster resources to authenticate against the cloud provider
 	BoundServiceAccountSigningKey String
+
+	// Kms Id for the bound service account signing key
```
Comment (zgalor) [inline model/clusters_mgmt/v1/sts_type.model:49]: The comment needs a '.' for swagger to properly generate documentation (also in line 46)

**Comment by zgalor** [comment]
Looks like you ran `commit --amend` instead of `commit` because this also includes @tzvatot 's previous commit

**Comment by zgalor** [comment]
> Related issue: https://issues.redhat.com/browse/SDE-1882

better link a story than an epic

**Pull request closed without merging**